_posts/2020-02-14-status-update.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "14th Feb 2020: IAC Team Status Update"
+author: criskoduck
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - tooling
+  - personal
+  - honeycomb
+  - metrics
+  - litmus
+  - windows
+  - powershell
+---
+
+The IAC team has been working on a lot of interesting things over the past week. Here's a brief overview of what we've been up to:
+
+## Updates to Puppet Modules
+We've released an update to one module since last week's update: 
+
+- [puppetlabs/pwshlib](https://forge.puppet.com/puppetlabs/pwshlib) (v0.4.1)
```

Comment:
Could probably use a brief overview for this in addition to scheduled task? Maybe just "to fix a bug causing runtime errors on older versions of ruby"?